x/ibc/testing/types.go

```diff
@@ -31,7 +31,7 @@ func (conn *TestConnection) AddTestChannel() TestChannel {
 // non-existent channel usually to test for its non-existence.
 func (conn *TestConnection) NextTestChannel() TestChannel {
 	portID := "transfer"
-	channelID := fmt.Sprintf("%s-%d", conn.ID, len(conn.Channels))
```

Comment:
@colin-axner I had to remove this because the handler tests were failing

Reply:
sounds good, it was only there for readability